### DISCUSSION TITLE
Artisan 2.8.2 won't start on Ubuntu 23.04

### DISCUSSION THREAD

**Discussion by wab0111** [artisan-roaster-scope/artisan#1163] [Q&A]
I purchased an inexpensive Gateway computer (11 inch, 4 GB ram and 64 GB storage) to install Artisan. The resources were so low, I replaced it with Ubuntu 23.04. I tried installing the deb file by downloading and double clicking for a simple install. The icon was present in the programs, but when I try to run the program, nothing happens other than opening the ubuntu desktop. I went to the Ubuntu Software app to uninstall but it wasn't in the list of installed programs. Then I searched for artisan in the Snap store, and installed that copy since it may be an approved (compatible version) with Ubuntu. It does show up in the list of installed programs now, but I still get the same result of trying to run the program and it just brings me to the Ubuntu desktop.

I also tried Ubuntu 22.04.2, but had same issues with trying to run Artisan and wifi card wasn't recognized.

It's been about 10-15 years since I used Ubuntu (on an acer netbook). Anyway, hoping to get some assistance here. I am planning to only run Artisan on this computer with a Hottop 2k+ roaster.

**Comment by MAKOMO** [comment]
Artisan 2.8.2 is known to fail to startup under Ubuntu 22.04 and 23.04 (Issue #1001). This has been fixed recently. The current Continuous builds should run on 23.04.

https://github.com/artisan-roaster-scope/artisan/releases

**Reply by wab0111** [reply]
Thanks. I found that it runs on Lubuntu 23.04 as well. I think part of the issue is that the resources may have been too low for Ubuntu. I couldn't seem to get it to work on Ubuntu 23.04 either, and it was pretty laggy. Lubuntu seems to be snappy and responsive.

**Reply by jedimaster2001** [reply]
Hi, have you tried 2.10.2 on any version of Ubuntu? (ubuntu, lbuntu, xbuntu)?

**Reply by wab0111** [reply]
I haven't. Since I've had it working on 2.8.2 on Lubuntu 23.04,  I haven't changed versions.
